protocol/streaming/full_node_streaming_manager.go

```diff
@@ -545,17 +536,36 @@ func getStreamUpdatesFromOffchainUpdates(
 func (sm *FullNodeStreamingManagerImpl) SendOrderbookUpdates(
 	offchainUpdates *clobtypes.OffchainUpdates,
 	blockHeight uint32,
-	execMode sdk.ExecMode,
+	ctx sdk.Context,
```

Comment:
might as well read block height from ctx? or are you worried about the +1 problem for check tx?

Reply:
Changed this function and couple of others to only take in `ctx`